cypress/TestCases/FireboltCertification/UserInterest.feature

```diff
@@ -0,0 +1,113 @@
+            | Without Interest type                                   | without interest type                                        |
+            | Invalid Interest type - integer params                  | with numeric interestType                                    |
+            | Invalid Interest type - test params                     | with string interestType                                     |
+            | Invalid Interest type - boolean params                  | with boolean interestType                                    |
+            | Invalid channelType in channel entity                   | with invalid channelType in channel entity                   |
+            | Boolean channelType in channel entity                   | with boolean channelType in channel entity                   |
+            | Integer channelType in channel entity                   | with integer channelType in channel entity                   |
+            | Invalid programType in program entity                   | with invalid programType in program entity                   |
+            | Boolean programType in program entity                   | with boolean programType in program entity                   |
+            | Integer programType in program entity                   | with integer programType in program entity                   |
+            | Invalid musicType in music entity                       | with invalid musicType in music entity                       |
+            | Boolean musicType in music entity                       | with boolean musicType in music entity                       |
+            | Integer musicType in music entity                       | with integer musicType in music entity                       |
+            | Invalid seriesId in program episode entity              | with invalid seriesId in program episode entity              |
+            | Invalid seasonId in program episode entity              | with invalid seasonId in program episode entity              |
+            | Invalid seriesId and seasonId in program episode entity | with invalid seriesId and seasonId in program episode entity |
+            | Invalid seriesId in program season entity               | with invalid seriesId in program season entity               |
+            | Invalid programType for program entity with seriesId    | with invalid programType for program entity with seriesId    |
+            | Invalid programType for program entity with seasonId    | with invalid programType for program entity with seasonId    |
+
+    @coreSDK @sdk @userinterest
+    Scenario Outline: Content.requestUserInterest - Positive Scenario: with type <Scenario>
+        And 1st party app invokes the 'Firebolt' API to 'get requestUserInterest with type <setParam>'
+        Then 'Firebolt' platform responds to '1st party app' with '<method_Content>'
+
```

Comment:
also, a little confused on below . Do we have any event onUserInterestedIn  ?? 
![image](https://github.com/rdkcentral/firebolt-certification-suite/assets/148241508/5946d2e4-65aa-4190-835b-82ec843b7330)

Reply:
I think this is typo.. Let me double check the flow